cmd/ledgerutil/main.go

```diff
@@ -41,30 +40,31 @@ func main() {
 		return
 	}
 
-	// Determine result json file location
-	var resultFilepath string
-	if outputDir != nil {
-		resultFilepath = *outputDir
-	} else {
-		resultFilepath, err = os.Getwd()
+	// Set results directory location; an empty location sets the current directory by default
+	if *outputDir != "" {
+		err = os.Chdir(*outputDir)
```

Comment:
I don't think that we should be changing the working dir.

Reply:
The reasoning for this change is the same as the comment above